tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -35,6 +35,9 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 USE_CHAIN_DMA=true,NUM_SDW_AMP_LINKS=2,SDW_SPK_STREAM=SDW1-Playback,SDW_SPK_IN_STREAM=SDW1-Capture,\
 SDW_DMIC_STREAM=SDW0-Capture"
 
+"cavs-sdw\;sof-mtl-rt713-l0-rt1316-l12-rt1713-l3\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1,\
+NUM_HDMIS=3,SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
```

Comment:
I though NUM_HDMIS was set to 3 as a platform default?

I don't know why some platforms has NUM_HDMIS=0 btw, this looks quite wrong now.

Reply:
> I though NUM_HDMIS was set to 3 as a platform default?
> 

Yes, and updated.

> I don't know why some platforms has NUM_HDMIS=0 btw, this looks quite wrong now.

Those are for MTL RVP, and i915 driver is not ready on the sof-dev branch. We will remove those NUM_HDMI=0 once i915 driver is ready.

Reply:
@bardliao @plbossart we do need to address this better. The HDA generic driver can handle missing HDMI and the SDW mach drivers should do the same. We can have variants with internal graphics disabled and this should not prevent audio from working on these devices (and we don't want to ship separate tplg variants for such cases).